packages/vue/src/components/SpTokenTransfer/SpTokenTransfer.vue

```diff
@@ -428,7 +428,7 @@ export default defineComponent({
 
       let payload: any = {
         amount,
-        toAddress: state.tx.toAddress,
+        receiver: state.tx.receiver,
```

Comment:
This `payload` obj can not be changed since outside code is expecting it to be of type [MsgSend](https://github.com/tendermint/vue/blob/832a377b244b2f8e361055f6eb00346a7379ad03/packages/template/src/store/generated/cosmos/cosmos-sdk/cosmos.bank.v1beta1/module/types/cosmos/bank/v1beta1/tx.ts#L9)